src/outlet/index.ts

```diff
@@ -224,11 +232,44 @@ export class Outlet {
 		const { tokenUrlName, tokenSecretName, tokenIdName, itemStorageKey } = this.tokenConfig
 
 		try {
-			const tokens = readMagicUrl(tokenUrlName, tokenSecretName, tokenIdName, itemStorageKey, this.urlParams)
+			const newToken = readMagicUrl(tokenUrlName, tokenSecretName, tokenIdName, itemStorageKey, this.urlParams)
```

Comment:
Could you please add a simple unit test for this function? Or move some of the logic out into a separate function to make it easier to add a unit test without mocking localStorage? 

Maybe something like: shouldTokenBeReplaced(oldToken, newToken)?

Reply:
Okay, I separated a new function (readTokenFromMagicUrl) and added unit test.
But readMagicUrl is not used anywhere now and readTokenFromMagicUrl is used only in readMagicLink function